### PR TITLE
feat: add some app configuration instructions

### PR DESCRIPTION
**Pull request by WilsontheWolf**
Adds a page to the userguide with some instructions to configure some apps.

### CODE REVIEW

**Comment by diederikdehaas** [comment]
GIMP has been [removed from Testing](https://tracker.debian.org/pkg/gimp) due to [bug 1078402](https://bugs.debian.org/1078402), that's why you **currently** don't see it (on the PineNote).  
The bug has been fixed, but only in the version in Experimental. Packages in Unstable transition to Testing, but not ones uploaded to Experimental. So for it to become available in Testing a new upload needs to be done (to Unstable), but I don't know when that happens.  
I'm going to **assume** it will happen before the Freeze starts so that it will be available when Trixie gets released as Stable.

File: userguide_raw/docs/apps.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Apps
+This page is dedicated to the instructions for getting/configuring various apps to work nicely on the PineNote.
+## GIMP
```
Comment (diederikdehaas) [inline userguide_raw/docs/apps.md:3]: Please add blank lines before and after headings.

Comment (WilsontheWolf) [reply userguide_raw/docs/apps.md:3]: Fixed

**Comment by diederikdehaas** [comment]
Please squash the commits (``git rebase -iS HEAD~2`` and follow the instructions) and force-push your branch.  
(And for next time you can do ``git commit -avS --amend``)

**Comment by m-weigand** [comment]
sorry for taking so long, somehow forgot about this PR and only noticed now. Thanks for taking the time to contribute!

**Pull request merged**